suite-native/discovery/src/utils.ts

```diff
@@ -1,38 +1,37 @@
 import { A, F } from '@mobily/ts-belt';
 
-import { NetworkCompatible, AccountType, NetworkSymbol } from '@suite-common/wallet-config';
+import { AccountType, Network, NetworkSymbol } from '@suite-common/wallet-config';
 import { Account } from '@suite-common/wallet-types';
 
-const NORMAL_ACCOUNT_TYPE = 'normal';
-
-// network uses undefined for normal type, but account uses 'normal'
-const normalizedAccountType = (accountType?: AccountType) => accountType ?? NORMAL_ACCOUNT_TYPE;
-
 export const getNetworksWithUnfinishedDiscovery = (
-    enabledNetworks: readonly NetworkCompatible[],
+    enabledNetworks: readonly Network[],
```

Comment:
This whole function used to filter out `NetworkCompatible`, now it filters `Network` with regard to its accounts (if there are some). So the collection length is different (smaller) - will it break something?
I think no, this is where it is used:
- `selectNetworksWithUnfinishedDiscovery`, and that is used:
  - `selectShouldRunDiscoveryForDevice` which [only looks if length > 0](https://github.com/trezor/trezor-suite/blob/5b29207bb5024babd48fd5ed504c3ca3b6ee51d7/suite-native/discovery/src/discoverySelectors.ts#L136-L138) :heavy_check_mark: 
  - `startDescriptorPreloadedDiscoveryThunk` which handles the foreach account :heavy_check_mark: